configs/vision/owkin/phikon/offline/camelyon16.yaml

```diff
@@ -0,0 +1,131 @@
+---
```

Comment:
we should add these configs in the tests, the `test_configuration_initialization` to test that the config can be parsed